detect: {

  #detection node 에서 사용하는 값들 설정하는 위치
  frame_id: "velodyne",

  sub_pc_topic: "/velodyne_points",
  sub_pc_angle: 10.0,
  sub_pc_queue_size: 1,

  use_voxelGrid_: false,
  voxel_size: 0.01,
  min_points_per_voxel: 1,

  pub_pcs_segmented_topic: "/segment/clouds_segmented",

  ## Important to use roi filter for "Ground remover"
  #use_roi_filter: false,
  use_roi_filter: true,

  ## Ground Segmenter
  # type: string
  # default: "GroundPlaneFittingSegmenter"
  ground_remover_type: "GroundPlaneFittingSegmenter",
  #ground_remover_type: "GroundRANSACSegmenter",
  ## Segment non-ground point cloud, otherwise, only Ground Segmenter
  # default: false
  #use_non_ground_segmenter: true,
  use_non_ground_segmenter: false,
  ## non-ground segmenter type
  # default: "RegionEuclideanSegmenter"
  non_ground_segmenter_type: "RegionEuclideanSegmenter",
  # non_ground_segmenter_type: "EuclideanSegmenter",
}
